File: app/src/main/java/com/example/dcct/model/RecordModel.java
package com.example.dcct.model;

import com.example.dcct.bean.BackResultData;
import com.example.dcct.bean.Record;

import java.util.List;

public interface RecordModel {
    void getAllQueryRecord(long id,ReturnQueryRecord returnQueryRecord);
    interface ReturnQueryRecord{
        void onComplete(BackResultData<List<Record>> listBackResultData);
        void onError(String msg);
    }
}

File: app/src/main/java/com/example/dcct/model/RecordModel.kt
package com.example.dcct.model

import com.example.dcct.bean.BackResultData
import com.example.dcct.bean.Record

interface RecordModel {
    fun getAllQueryRecord(id: Long, returnQueryRecord: ReturnQueryRecord)
    interface ReturnQueryRecord {
        fun onComplete(listBackResultData: BackResultData<List<Record>>)
        fun onError(msg: String)
    }
}